frontend/wasm/src/editor_api.rs

```diff
@@ -122,10 +122,13 @@ impl JsEditorHandle {
 					} else {
 						js_sys::Float64Array::default()
 					};
-					updateImage(image.path, image.mime, &image.image_data, transform, document_id);
+					updateImage(image.path, image.node_id, image.mime, &image.image_data, transform, document_id);
 				}
 				#[cfg(feature = "tauri")]
-				fetchImage(image.path.clone(), image.mime, document_id, format!("http://localhost:3001/image/{:?}_{}", &image.path, document_id));
+				{
+					let identifier = format!("http://localhost:3001/image/{:?}_{}", &image.path, document_id);
+					fetchImage(image.path.clone(), image.node_id, image.mime, document_id, identifier);
```

Comment:
some nitpicking:
- `.clone` isn't needed here, `image.path` can be consumed, it isn't needed later
- `&image.path`: the `&` isn't needed. All rust `format`-macros auto-ref their arguments, so they never consume later needed variables.

Reply:
Sure, this was just copied from here before.